src/main/resources/config/webserver.yml

```diff
@@ -1,4 +1,5 @@
-base: C:\\edc\\demo-2
-transferMinSize: 100
+base: C:/DEV/
 indexUrlEnabled: true
+transferMinSize: 100
+requestMaxSize: 2GB
```

Comment:
50Mo by default and no more for the requestMaxSize

Reply:
in .yml is not the default value, it's a loaded value

Reply:
the default value is in WebServerConfig.java (line 18)private long requestMaxSize = (long) 20E3;